.projenrc.js

```diff
@@ -50,8 +50,8 @@ const project = new AwsCdkConstructLibrary({
   // authorUrl: undefined,                                                     /* Author's URL / Website. */
   // autoDetectBin: true,                                                      /* Automatically add all executables under the `bin` directory to your `package.json` file under the `bin` section. */
   // bin: undefined,                                                           /* Binary programs vended with your module. */
-  bundledDeps: ['aws-lambda-ses-forwarder', 'aws-sdk', 'aws-lambda', '@types/aws-lambda'], /* List of dependencies to bundle into this module. */
-  deps: ['@seeebiii/ses-verify-identities@3.0.7'], /* Runtime dependencies of this module. */
+  bundledDeps: ['aws-lambda-ses-forwarder', 'aws-sdk', 'aws-lambda', '@types/aws-lambda', '@seeebiii/ses-verify-identities@3.0.7'], /* List of dependencies to bundle into this module. */
```

Comment:
The dependencies `aws-lambda-ses-forwarder`, `aws-sdk` and `aws-lambda` belong to the Lambda function that is forwarding the emails. Since you bundle this Lambda function now (instead of letting `NodejsFunction` bundle it), I believe we could even remove them from here. (Because the dependencies are now bundled in the Lambda function artifact) But fine for now, I can clean it up afterwards :)

Reply:
I see that you may have forgotten to merge this, so I've gone ahead and looked into this very quickly.

Sadly, we encounter [this error](https://github.com/aws/jsii/blob/main/packages/jsii/lib/project-info.ts#L372) by removing these bundled dependencies, because these packages are not jsii-enabled.

I have however shifted `@types/aws-lambda` into the development dependencies (although this didn't actually make any tangible difference to the artifact size, unfortunately!).